src/test/suite/extension.test.ts

```diff
@@ -119,4 +122,23 @@ suite('Verification Gutter', () => {
     assert.deepStrictEqual([ new vscode.Range(7, 1, 8, 1) ], ranges.get(2));
     assert.deepStrictEqual([ new vscode.Range(3, 1, 3, 1), new vscode.Range(5, 1, 5, 1) ], ranges.get(0));
   });
+});
+
+suite('commands', () => {
+  test.skip('restart server', async () => {
+    const program = `
+method Foo(x: nat) returns (y: nat) 
+  ensures y > 2;
+{ 
+  return x + 2; 
+}`;
+    const extension = vscode.extensions.getExtension('dafny-lang.ide-vscode')!;
+    await extension.activate();
+    const document = await vscode.workspace.openTextDocument({ language: 'dafny', content: program });
+    const symbols1 = await vscode.commands.executeCommand('vscode.executeDocumentSymbolProvider', document.uri) as DocumentSymbol[];
+    await vscode.commands.executeCommand('dafny.restartServer');
```

Comment:
Why do you need to restart the server here? Are you only checking if the server returns the same commands?

Reply:
This test is for testing the restartServer command, so it should execute it right? It was correctly identifying the bug that this PR fixes, but sadly it also causes the subsequent tests to fail, so I can't merge it for now.